src/linear_algebra/finite_dimensional.lean

```diff
@@ -126,9 +124,8 @@ by { casesI nonempty_fintype K, haveI := fintype_of_fintype K V, apply_instance
 variables {K V}
 
 /-- If a vector space has a finite basis, then it is finite-dimensional. -/
-lemma of_fintype_basis {ι : Type w} [fintype ι] (h : basis ι K V) :
-  finite_dimensional K V :=
-⟨⟨finset.univ.image h, by { convert h.span_eq, simp } ⟩⟩
+lemma of_fintype_basis {ι : Type w} [_root_.finite ι] (h : basis ι K V) : finite_dimensional K V :=
```

Comment:
Should it be called `of_finite_basis`?

Reply:
Yeah, but I didn't rename it because `of_finite_basis` already exists. If you have a better naming scheme than `_fintype`, `_finite`, let me know.